config/manager/kustomization.yaml

```diff
@@ -6,3 +6,23 @@ images:
 - name: controller
   newName: quay.io/operator-framework/operator-controller
   newTag: devel
+replacements:
+- source: # replaces UNPACK_IMAGE in manager.yaml with image set by kustomize above
```

Comment:
I prefer doing it this way because it ensures that `unpack-image` will always equal the image for `manager` without needing to write out the `quay` image more than once. I feel it's more maintainable if we ever change that url or do some custom deployment; we wouldn't have to change it in multiple places. 

Also, IIRC we wanted to eventually remove the `sed "s/:devel/:$(VERSION)/g"` since it's a bit brute-forcey, and if we did then these two image strings would still match without any need to do anything else.